### PR TITLE
FEATURE: Show all categories in composer

### PR DESCRIPTION
**Pull request by CvX**
…and just prioritize the current one, instead of hiding other categories.

Context: when you open the composer by clicking "New Topic" button when in a category, or by clicking "New Topic" in the share-popup, the category selector shows only the current category and its children (and "Uncategorized"). You can still find other categories, but you have to search by name.
This PR changes that, so you now can see all the categories in the dropdown, and those that are relevant (again: current, children and uncategorized) are displayed before all other categories.

tldr: don't make choosing other categories harder - make choosing relevant ones easier.

Before/After

before after

There are only exactly three categories in the first screenshot. The second one has all of them (notice the scroll)

### CODE REVIEW

**Review by eviltrout — APPROVED** [review]
Let's give it a try!
